examples/eris/src/index.js

```diff
@@ -1,14 +1,32 @@
-const Eris = require('eris');
+const Eris = require("eris");
+const fs = require("fs");
```

Comment:
```suggestion
const fs = require("node:fs");
```

https://nodejs.org/api/modules.html#core-modules